modules/QnA/resources/qnaService.js

```diff
@@ -192,6 +192,9 @@
             else if (this.getType() === "AnswerOnAir"){
                 return gM('qna-answer-on-air');
             }
+            else if (this.cuePoint().overrideModeratorName !== false){
```

Comment:
@MaksymKorniiov  
I don't know why but this looks fishy. This function is some kind of a translation function and it handles content by 'this.getType()'. Your addition to this function doesn't look like the same syntax and I don't think it will reach the default  this.cuePoint().userId. Maybe we can write this in a different way?